yarn.lock

```diff
@@ -5832,8 +5832,8 @@ node-modules-regexp@^1.0.0:
   resolved "https://registry.yarnpkg.com/node-modules-regexp/-/node-modules-regexp-1.0.0.tgz#8d9dbe28964a4ac5712e9131642107c71e90ec40"
   integrity sha1-jZ2+KJZKSsVxLpExZCEHxx6Q7EA=
 
-node-notifier@^6.0.0:
-  version "6.0.0"
+node-notifier@^8.0.1:
```

Comment:
We should find which library is using this dependency and we will upgrade it if it is possible. It will be more consistent.

Reply:
I've searched the project but node-notifier hasn't showed up in the node_modules.